db/schema.rb

```diff
@@ -10,9 +10,9 @@
 #
```

Comment:
This file shouldn't be changed

Reply:
This is done automatically when running the migration. Because it requires adding the new `standard` field

Reply:
Do you want I manually remove unnecessary changes (like the `t0.`)?

Reply:
Yes please!

Reply:
Done